airbyte-webapp/src/pages/routes.tsx

```diff
@@ -53,7 +58,15 @@ const MainViewRoutes: React.FC = () => {
               <Route index element={<DestinationOverviewPage />} />
             </Route>
           </Route>
-          <Route path={`${RoutePaths.Source}/*`} element={<SourcesPage />} />
+          <Route path={RoutePaths.Source}>
+            <Route index element={<AllSourcesPage />} />
+            <Route path={SourcePaths.NewSource} element={<CreateSourcePage />} />
+            <Route path={SourcePaths.NewConnection} element={<CreateConnectionPage />} />
+            <Route path={SourcePaths.Root} element={<SourceItemPage />}>
+              <Route index element={<SourceOverviewPage />} />
+              <Route path={SourcePaths.Settings} element={<SourceSettingsPage />} />
+            </Route>
+          </Route>
```

Comment:
Same work as cloud routes

Reply:
@edmundito didn't you want to keep these nested routes in a separate file? https://github.com/airbytehq/airbyte/pull/20845#discussion_r1061328629

I don't really feel strongly either way, just as long as we're consistent. I have it on my to-do list to refactor the destination paths like you did in https://github.com/airbytehq/airbyte/pull/20845, so that's why I'm wondering how we want to proceed here